game.py

```diff
@@ -24,7 +24,7 @@ def play_game(self):
       self.players.append(Player('x'))
       while True:
         self.board.disp_board()
-        try:
+        try:  
```

Comment:
The try/except block should be around just the bits of code that you know should be raising exceptions. In this case, around the add_piece method. Put this in a while loop so that if there is an error it just keeps asking for input.

If there is an error in some other method, you WANT the program to crash (so you can find the bug)